app/javascript/components/CSVUploadForm/index.js

```diff
@@ -0,0 +1,27 @@
+const SELECTORS = {
+  fileInput: '#csv_upload_form_file'
+};
+
+class CSVUploadForm {
+  constructor(csvUploadForm) {
```

Comment:
We usually use `elementRef` as the constructor argument 
```suggestion
  constructor(elementRef) {
```
![Screen Shot 2021-07-14 at 6 12 22 PM](https://user-images.githubusercontent.com/6076762/125613112-480b8242-5c63-4fe3-8279-bf74c07861ee.png)

Reply:
Fixed :)